src/pages/messages/components/NavigationIconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { IconButton, IconButtonProps } from '@mui/material';
+import { Link, LinkProps } from 'react-router-dom';
+
+interface NavigationIconButtonProps extends IconButtonProps, Pick<LinkProps, 'to'> {
+  title: string;
+  navigateTo: 'previous' | 'next';
+}
+
+export const NavigationIconButton = ({ title, navigateTo, sx, ...rest }: NavigationIconButtonProps) => (
+  <IconButton
+    component={Link}
+    title={title}
```

Comment:
`title` ligger vel i `IconButtonProps` også, så kan bruke defaults der
```suggestion
  navigateTo: 'previous' | 'next';
}

export const NavigationIconButton = ({ navigateTo, sx, ...rest }: NavigationIconButtonProps) => (
  <IconButton
    component={Link}
```

Reply:
Litt vanskelig å se i diffen, men jeg har bare fjernet `title` som custom prop